public/css/stars.css

```diff
@@ -32,13 +34,18 @@
     top: 20%;
     left: 20%;
     background: transparent url(../img/stars.png) repeat top center;
+    background-attachment: fixed;
     z-index: -1;
     animation: move-stars-two 10s linear infinite;
     -ms-animation: move-stars-two 10s linear infinite;
     -moz-animation: move-stars-two 10s linear infinite;
     -webkit-animation: move-stars-two 10s linear infinite;
 }
 
+#bkg{
+    background-attachment: fixed;
+    position: fixed;
+}
```

Comment:
Don't need this, apply the styles directly on the div with class `fixed` and you're already adding the `background-attachment: fixed` on the stars above.

Reply:
I will do it right away